src/settings/index.js

```diff
@@ -533,28 +533,32 @@ class _Settings extends React.Component {
         )
   }
 
+  chainButton (chainId, name) {
```

Comment:
can we turn this into a component that takes `isSelected` as prop? think that would be more stylistically consistent, then we can just check the current chain in the store once in the calling code and set `isSelected` based on this comparison

Reply:
I would have done this already but the nomenclature collision with styled components above made this the path of least resistance.  However since you've called it I'll rename all the styled components.

Reply:
Ok done, not sure it's better looking code but there's more of it so there's definitely an increased sense of value.

Reply:
I know in the past you've expressed a preference to extract things out into components as it's more of a "React-type" style and I agree, so from that perspective I do think this looks good.